ferretdb/ferretdb.go

```diff
@@ -229,20 +234,24 @@ func (f *FerretDB) MongoDBURI() string {
 
 var (
 	loggerOnce sync.Once
-	logger     *zap.Logger
+	logger     *slog.Logger
 )
 
 // getGlobalLogger retrieves or creates a global logger using
 // a loggerOnce to ensure it is created only once.
-func getGlobalLogger() *zap.Logger {
+func getGlobalLogger() *slog.Logger {
 	loggerOnce.Do(func() {
-		level := zap.ErrorLevel
+		level := slog.LevelError
 		if version.Get().DebugBuild {
-			level = zap.DebugLevel
+			level = slog.LevelDebug
 		}
 
-		logging.Setup(level, "console", "")
-		logger = zap.L()
+		opts := &logging.NewHandlerOpts{
+			Base:  "console",
+			Level: level,
+		}
+		logging.Setup(opts, "")
+		logger = slog.Default()
 	})
 
 	return logger
```

Comment:
I think we should remove that code and just use `slog.Default()` if `SLogger` is nil

Reply:
Sure, I think `version.Get().DebugBuild` is left over from when there was no option to configure a logger.

Reply:
Just thinking, if `SLogger` is nil, we still want to call `logging.Setup` to populate `RecentEntries`. Do you recall why `sync.Once` is used to setup logger though?

Reply:
`sync.Once` is used because setup functions change the global state: change the default logger, change what is being used by old `log` package in code like `log.Printf`